target/linux/ath79/dts/ar9344_fortinet_ap-dual.dtsi

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+#include "ar934x_fortinet_loader.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
```

Comment:
Do you plan to add more devices using this dtsi file in the near future? 

If you do not plan this why did you split this into multiple dtsi files?

Reply:
I don't have another similar device right now, but there are similar ones out there which I may have my hands on in the future, or someone else can add them.

I generally modeled the DTS and DTSIs after how it is for the similar Engenius devices which are equivalent hardware but different flash layout

like this commit
513f9855e9ce710c379cb3af4cd86b899f637c6e

Reply:
ok this is fine with me.